app/renderer/js/components/webview.ts

```diff
@@ -7,6 +7,7 @@ import ConfigUtil = require('../utils/config-util');
 import SystemUtil = require('../utils/system-util');
 import BaseComponent = require('../components/base');
 import handleExternalLink = require('../components/handle-external-link');
+import DomainUtil = require("../utils/domain-util");
```

Comment:
Nitpicking here, but we use single quotes throughout the app. 
I think we've missed a test in `xo`, I'll check this asap.

Reply:
corrected it !